ui/src/main/java/edu/wpi/grip/ui/pipeline/source/SwitchableSourceControlsView.java

```diff
@@ -0,0 +1,71 @@
+package edu.wpi.grip.ui.pipeline.source;
+
+
+import com.google.common.eventbus.EventBus;
+import edu.wpi.grip.core.SwitchableSource;
+import edu.wpi.grip.core.util.ExceptionWitness;
+import edu.wpi.grip.ui.util.DPIUtility;
+import javafx.scene.control.ToggleButton;
+import javafx.scene.control.Tooltip;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import org.controlsfx.control.SegmentedButton;
+
+import java.io.IOException;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * Provides controls for a {@link SwitchableSource}
+ */
+public class SwitchableSourceControlsView extends SourceControlsView<SwitchableSource> {
```

Comment:
Since we have plans to (sooner or later) make the multi image source possible have a drop down instead of arrows, maybe it would be better to have a specific `ImageSourceView` so we can add features to both the view and the source without changing the abstract class.